packages/strapi/bin/strapi-new.js

```diff
@@ -54,6 +54,8 @@ module.exports = function (name, cliArguments) {
       return process.exit(1);
     }
 
+    scope.dbforce = cliArguments.dbforce !== undefined;
```

Comment:
You have to also remove this line.
We not use this variable anywhere in this file.

Reply:
But `scope.dbforce` is used in `connectivity.js` to get the value of the CLI Argument. If I don't set it here, is there a way that I can directly access this argument in `connectivity.js` file?

Reply:
Ho yap 🤦‍♂️ my bad.